selfdrive/manager/manager.py

```diff
@@ -21,6 +22,10 @@
 
 
 def manager_init():
+
+  # update system time from panda
+  set_time(cloudlog)
```

Comment:
Should we also put a call in the launch script just before starting the agnos/neos updater?

Reply:
think it’s best to keep as much in manager as possible, there’s no logging or error handling in the launch script

Reply:
Would be nice to sync the time before trying to download an OS update though. AGNOS updater will retry until it succeeds, but think the NEOS downloader will just fail and prompt you to reboot if you start it too quickly.